pkg/apis/pingcap/v1alpha1/tidbcluster_component.go

```diff
@@ -118,7 +119,10 @@ func (a *componentAccessorImpl) PodManagementPolicy() apps.PodManagementPolicyTy
 	// unified podManagementPolicy check to avoid check everywhere
 	if policy == apps.OrderedReadyPodManagement {
 		return apps.OrderedReadyPodManagement
+	} else if policy == apps.ParallelPodManagement {
+		return apps.ParallelPodManagement
 	}
+	klog.Warningf("PodManagementPolicy \"%s\" is invalid. Fall back to use \"Parallel\" instead\n", policy)
```

Comment:
Usually, we do not add log in this file, as you can see, no klog imported in this file, each function will be called repeatedly during each sync which will flood too many logs, what about adding descriptions in types.go to describe this behavior clearly.

Reply:
@DanielZhangQD Thank you for your reply. Yes, I think it makes sense not to put the log in this file. But I suppose we can add an `enum` for kubebuilder in types.go to prevent invalid input. I updated the code in the latest commit.